osbuild/pipeline.py

```diff
@@ -121,6 +122,7 @@ def prepare_arguments(self, args, location):
         args["options"] = self.options
         args["meta"] = meta = {
             "id": self.id,
+            "manifest": self.manifest,
```

Comment:
Yea, it stuffs the entire manifest into the 'meta' part of the arguments that are given to all stages. It's a pretty naive way of doing this and it does allow for someone to directly use the manifest instead of only the options given to the specific stage.

My alternative is/was to introduce something not-a-stage, so a new type of module that would only receive the entire manifest.